src/main/java/com/sandvoxel/generitech/common/tileentities/machines/TileEntityPulverizer.java

```diff
@@ -234,120 +249,116 @@ public void update() {
             }
         }
 
+        //for taking power/fuel from the block
         if ((this.container.takePower(powerUsage, true) == powerUsage && machineTier != MachineTier.TIER_0) || fuelRemaining > 0) {
             if (machineActive && !pulverizerPaused) {
                 ticksRemaining--;
 
                 if (machineTier != MachineTier.TIER_0)
                     this.container.takePower(powerUsage, false);
-                if (fuelRemaining > 0)
-                    fuelRemaining--;
+
 
             }
+        }
+        if (fuelRemaining > 0)
+            fuelRemaining--;
 
-            if (inventory.getStackInSlot(0) != null && inventory.getStackInSlot(1) == null) {
-                ItemStack itemIn = inventory.getStackInSlot(0);
-                ItemStack itemOut;
 
-                if (!PulverizerRegistry.containsInput(itemIn))
-                    return;
+        //code for item in
+        if (inventory.getStackInSlot(0) != null && inventory.getStackInSlot(1) == null) {
+            ItemStack itemIn = inventory.getStackInSlot(0);
+            ItemStack itemOut;
 
-                if (itemIn.stackSize - 1 <= 0) {
-                    itemOut = itemIn.copy();
-                    itemIn = null;
-                } else {
-                    itemOut = itemIn.copy();
+            if (!PulverizerRegistry.containsInput(itemIn))
+                return;
 
-                    itemOut.stackSize = 1;
-                    itemIn.stackSize = itemIn.stackSize - 1;
-                }
+            if (itemIn.stackSize - 1 <= 0) {
+                itemOut = itemIn.copy();
+                itemIn = null;
+            } else {
+                itemOut = itemIn.copy();
 
-                if (itemIn != null && itemIn.stackSize == 0) itemIn = null;
-                if (itemOut.stackSize == 0) itemOut = null;
+                itemOut.stackSize = 1;
+                itemIn.stackSize = itemIn.stackSize - 1;
+            }
 
-                inventory.setInventorySlotContents(0, itemIn);
-                inventory.setInventorySlotContents(1, itemOut);
+            if (itemIn != null && itemIn.stackSize == 0) itemIn = null;
+            if (itemOut.stackSize == 0) itemOut = null;
 
-                ticksRemaining = 200;
-                machineActive = true;
+            inventory.setInventorySlotContents(0, itemIn);
+            inventory.setInventorySlotContents(1, itemOut);
 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1125)